File: solutions_11-20/Euler_20.py
## find the sum of the digits in !100 ? ##

import math
import timeit

start = timeit.default_timer()

print "Answer: %s" % sum([int(i) for i in str(math.factorial(100))])
stop = timeit.default_timer()
print "Time: %s" % str(stop - start)

File: Perl/perl_solutions_11-20/Euler_20.pl
use strict;
use warnings;
use Time::HiRes qw( clock );
use Math::BigInt;


sub hundredfac_sum
{
    my $total = 0;
    my $hundredstr = "100";
    my $hundred = Math::BigInt->new($hundredstr);
    $hundred->bfac();
    foreach my $i (split(//, "$hundred")) {
        $total += $i;
    }
    return $total;
}

my $start = clock();
my $answer = hundredfac_sum();
my $stop = clock();
print "Answer: $answer\n";
printf "Time: %.5f\n", $stop -$start;
